core/sail/shacl/src/main/java/org/eclipse/rdf4j/sail/shacl/ShaclSail.java

```diff
@@ -967,10 +967,61 @@ public IsolationLevel getDefaultIsolationLevel() {
 
 	public static class TransactionSettings {
 
+		public enum PerformanceHint implements TransactionSetting {
```

Comment:
Given the last-minute nature, let's mark this with an `@Experimental` tag.

Reply:
Will do.